config/index.js

```diff
@@ -1,17 +1,57 @@
 import { get } from 'lodash';
+import { error, debug } from 'heroku-logger';
+import { Assert } from '@sindresorhus/is';
 
 import app from './app';
+import auth from './auth';
 import database from './database';
 import filesystem from './filesystem';
 
+/**
+ * Load an environment variable.
+ *
+ * @param {String} name
+ * @param {Assert} validator
+ * @param {*} default
+ */
+export function env(name, validator = null, defaultValue = null) {
```

Comment:
Dig the idea of validator! Just curious if the `defaultValue` being the second parameter makes more sense 🤔 Did you find that's not the case because in the instances where you use `env()` no defaults were being passed and it's more about ensuring the env variable is actually defined per env rather than relying on a default? Just curious on the train of thought.

Reply:
Yeah, aside from `PORT`, I don't think it's all that useful to have a default value for these (outside of the "default" set in the `.env.example` file that folks use as their template).

I think default values that we set here in `config` also reduce the helpfulness of validation. For example, if we default `APP_URL` to `http://localhost:3000`, we'd never get an error if it's unset, and instead we'd just silently default production environments to the wrong (but still technically "valid") value.

Reply:
Ya, agreed. That's definitely been an issue in the past. This does seem to help act as a mediator to ensure env variables are checked in one location rather than needing to do each check in the place they're ultimately used.

Reply:
It's decided! Removed the ability to set defaults here in 43e2c52. I also added some code (again, inspired by Laravel) to parse native types for integers and booleans in ed5ea4f.